build.csx

```diff
@@ -153,7 +153,7 @@ targets.Add(
 
 targets.Add(
     "pack",
-    DependsOn("build", "outputDirectory"),
+    DependsOn("build", "pdbgit"),

```

Comment:
Yeah, that was what my first comment intended, but then I noticed that it wasn't _required_. Still, having the dependency is intention-revealing.